frontend/src/index.scss

```diff
@@ -0,0 +1,21 @@
+@import "~@openedx/paragon/scss/core/_functions.scss";
+@import "~@openedx/paragon/scss/core/_variables.scss";
+@import "~bootstrap/scss/mixins";
+//@import "~bootstrap/scss/root";
+//@import "~bootstrap/scss/reboot";
+//@import "~@openedx/paragon/scss/core/typography";
+//@import "~@openedx/paragon/scss/core/grid";
+// The react-loading-skeleton package.json defines an 'exports' field which specifies a '.css'
+// extension on this export.  Without the '.css' on the end of this line, webpack can't find the file
+// and the build fails.
+// See https://webpack.js.org/guides/package-exports/ for more information on how webpack uses
+// the 'exports' field.
+//@import "~react-loading-skeleton/dist/skeleton.css";
+//@import "~bootstrap/scss/transitions";
+//@import "~@openedx/paragon/scss/core/utilities";
+//@import "~bootstrap/scss/media";
+//@import "~bootstrap/scss/list-group";
+//@import "~bootstrap/scss/print";
+
```

Comment:
We don't need these utilities for our purposes. However, I keep them here so they can just be toggled in the future and diffing against paragon is easier.